flickr.py

```diff
@@ -34,7 +34,7 @@ def search_flickr_by_tags(tags):
         "method": "flickr.photos.search",
```

Comment:
add an if-else statement before params_diction, use type of search_term to determine what search method to use.

Reply:
alternatively, make search_term into a dictionary and pass dictionary[item] into search method.